src/NServiceBus.Core/ObjectBuilder/LightInject.g.cs

```diff
@@ -26,10 +26,11 @@
     http://twitter.com/bernhardrichter
 ******************************************************************************/
 
-// how are the conditional compilation symbols set?
```

Comment:
The comment I changed was added by @timbussmann when he wasn't sure how to handle the defines. Since what he had there was the best way to handle it, I changed the comment to indicate that vs it being a question.